packages/utilities/src/colors.ts

```diff
@@ -229,21 +229,26 @@ export function updateH(color: IColor, h: number): IColor {
 }
 
 export function updateA(color: IColor, a: number): IColor {
-  return assign({}, color, {
-    a: a,
-    str: (a === 100) ? `#${color.hex}` : `rgba(${color.r}, ${color.g}, ${color.b}, ${a / 100})`
-  });
+  return {
+    ...color, ...(() => {
+      return {
+        a: a,
+        str: (a === 100) ? `#${color.hex}` : `rgba(${color.r}, ${color.g}, ${color.b}, ${a / 100})`
+      };
+    })
+  };
 }
 
-function _numberToPaddedHex(num: number) {
+function _numberToPaddedHex(num: number): string {
   const hex = num.toString(16);
 
   return hex.length === 1 ? '0' + hex : hex;
 }
 
-function _named(str: string) {
+function _named(str: string): IRGB | undefined {
+  /* tslint:disable */
```

Comment:
I had to add this to get around the tslint error, which complains about 
(COLOR_VALUES as any) of using 'any'... 
Do you know how to fix it?

Reply:
can you try just removing the any? or COLOR_VALUES as {}
it should infer the typing from the file... hopefully....

Reply:
That won't work. It will then complain about you not having an indexor. Typescript says that if you want to access something using a string liter then you need to explicitly state that it can be accessed that way. If you want you can add [key: string]: string to COLOR_VALUES and it should be happy.